.github/workflows/analytics-tests.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     runs-on: blacksmith-2vcpu-ubuntu-2204
     steps:
       - uses: actions/checkout@v2
+      - name: Setting up 
+        run: sed -i 's/analytics-site\///' ./analytics-site/.eslintrc.json
```

Comment:
These are a no-op because the eslintrc.json file doesn't contain that text. Same goes for the other sites after I fixed the eslint

Reply:
Nice catch, updated :checkered_flag: